examples/OpenTracing.Examples/CommonRequestHandler/README.md

```diff
@@ -0,0 +1,32 @@
+# Common Request Handler example.
+
+This example shows an `ISpan` used with `RequestHandler`, which is used as a middleware (as in web frameworks) to manage a new `ISpan` per operation through its `BeforeRequest()`/`AfterResponse()` methods.
+
+Since its methods are not guaranteed to be run in the same thread, activation of such `ISpan`s is not done.
+
+```cs
+public void BeforeRequest(Object request, Context context)
+{
+    // we cannot use active span because we don't know in which thread it is executed
```

Comment:
Hmm.. that's no problem in C# with our `AsyncLocalScopeManager`. It correctly flows the scope, no matter on which thread the code is continued.